datafusion/optimizer/Cargo.toml

```diff
@@ -43,6 +43,7 @@ arrow = { workspace = true }
 chrono = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
+datafusion-functions-window = { workspace = true }
```

Comment:
this is something we have been trying to avoid -- explicit dependencies on functions in the optimizer (this allows faster compilation times)

Since the package is only used in tests, can you please move it into the `dev-dependencies` section below ?

Reply:
Done.